eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/EntityMangerFactoryServiceLocatorPass.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Compiler\ServiceLocatorTagPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+final class EntityMangerFactoryServiceLocatorPass implements CompilerPassInterface
```

Comment:
I :+1: this typo :smile: 

```suggestion
final class EntityManagerFactoryServiceLocatorPass implements CompilerPassInterface
```

Reply:
Yup, thanks for noticing. I should code less in the evenings ;)

Reply:
e58374f